include/domain_bridge/qos_options.hpp

```diff
@@ -121,20 +121,47 @@ class QosOptions
    */
   DOMAIN_BRIDGE_PUBLIC
   QosOptions &
-  lifespan(const std::int64_t & lifespan);
```

Comment:
Are ``deadline`` and ``lifespan`` meant to be changed ? Maybe we can leave these as const references then.

Reply:
There's no reason to pass integers as `const &`.
I think it's a good idea to change it now, it shouldn't affect domain bridge users.